screenly-cast/theme/screenly-cast/category_style.php

```diff
@@ -0,0 +1,439 @@
+<?php header('Content-type: text/css'); 
```

Comment:
It's a wrong way to generate CSS in this way. Maybe it's better use common CSS-file for render a single post and category's slider, and control view through JS